src/Plugin/Filter/EntityEmbedFilter.php

```diff
@@ -102,13 +104,24 @@ public function process($text, $langcode) {
             }
 
             $access = $entity->access('view', NULL, TRUE);
-            $access_metadata = CacheableMetadata::createFromObject($access);
-            $entity_metadata = CacheableMetadata::createFromObject($entity);
```

Comment:
We can remove the second line because we are rendering the embedded entity, that will itself include the cacheability metadata of the entity.